examples/get-started/first_figure.py

```diff
@@ -0,0 +1,159 @@
+"""
+1. Making your first figure
+===========================
+
+This tutorial page covers the basics of creating a figure. It will only use
+the ``coast`` module for plotting. Later examples will address other PyGMT
+modules.
+"""
+
+###############################################################################
+# Setting up the development environment
+# --------------------------------------
+#
+# PyGMT can be used in both a Python script and a notebook environment, such
+# as Jupyter. The tutorial's recommended method is to use a notebook, and the
+# code will be for a notebook environment. If you use a Python script instead,
+# there will be a difference in how the figure is shown that is explained in
+# the first example.
```

Comment:
> If you use a Python script instead, there will be a difference in how the figure is shown that is explained in the first example.

Please see my comment about `fig.show(method="external)` below.

Reply:
Fixed in  2024b6c